mirage/lE.ml

```diff
@@ -0,0 +1,179 @@
+(* (c) Hannes Menhert *)
+
+type configuration = {
+  email : Emile.mailbox option;
+  certificate_seed : string option;
+  certificate_key_type : X509.Key_type.t;
+  certificate_key_bits : int option;
+  hostname : [ `host ] Domain_name.t;
+  account_seed : string option;
+  account_key_type : X509.Key_type.t;
+  account_key_bits : int option;
+}
+
+module HTTP : Letsencrypt__HTTP_client.S with type ctx = Http_mirage_client.t (* FIXME *) =
```

Comment:
What is the FIXME about?

Reply:
Currently, we can fix into this PR what I have in my mind. The `HTTP_client.S` is only available by the double-underscore made by `dune`. It will be nice to write: `Letsencrypt.HTTP_client.S`.